package.json

```diff
@@ -51,5 +51,8 @@
     "ts-node": "~7.0.0",
     "tslint": "~6.0.0",
     "typescript": "3.9.7"
+  },
+  "dependencies": {
```

Comment:
This should be part of the `packages/engine/vscode/package.json` instead except if you added unit tests that require to have the dependencies in the global project.